extensions/3DTILES_extent/README.md

```diff
@@ -0,0 +1,136 @@
+# 3DTILES_extent
+
+## Contributors
+
+* Sean Lilley, [@lilleyse](https://github.com/lilleyse)
+* Samuel Vargas [@Samulus](https://github.com/Samulus)
+
+## Contents
+
+* [Overview](#overview)
+* [Defining Extents](#defining-extents)
+* [Holes in Extents](#holes-in-extents)
+* [Arc Types](#arc-types)
+* [Coordinate System](#coordinate-system)
+
+## Overview
+
+The extent 3dtiles extension allows the user to annotate the existence of a smaller 2D region(s) (an extent) in a given `tileset.json` via a 2D array of latitude, longitude pairs. This is useful for a variety of scenarios such as: Overlaying high resolution geometry ontop of low level geometry, insetting one tileset into another tileset, or clipping excess geometry inside of a provided tileset.
+
+## Defining Extents
+
+An extent is a collection of latitude and longitude coordinate pairs. Extents are two dimensional in nature, but an optional third component can be specified for **each** extent coordinate to specify its height (in meters). The coordinate pairs should be provided in **counterclockwise** winding order. Multiple extents can be specified. Convex and concave extents are both supported. At least three coordinates must be provided for an extent to be valid.

```

Comment:
`its height (in meters)` - more formally `its height (in meters) from the WGS84 ellipsoid`